code/datums/gamemode/role/role.dm

```diff
@@ -105,6 +105,8 @@
 	faction=fac
 	if(!faction)
 		ticker.mode.orphaned_roles += src
+	else
+		faction.members += src
```

Comment:
Highly illegal

Reply:
???
If they are linked to a faction why not add them to their members